include/klee/Core/TerminationTypes.h

```diff
@@ -28,7 +28,7 @@
   TTYPE(BadVectorAccess, 12U, "bad_vector_access.err")                         \
   TTYPE(Free, 13U, "free.err")                                                 \
   TTYPE(Model, 14U, "model.err")                                               \
-  TTYPE(Overflow, 15U, "overflow.err")                                         \
+  TTYPE(UndefinedBehavior, 15U, "undefined_behavior.err")                      \
```

Comment:
Please change this back. If required, feel free to add new categories.
"UndefinedBehaviour" is too broad. Moreover, many other errors ("Ptr" - aka out-of-bounds access, "Free" aka use-after-free) are undefined behaviour as well. I would like to keep the categories separated if possible.